### DISCUSSION TITLE
How to debug/test ldap connection?

### DISCUSSION THREAD

**Discussion by cjs1976** [requarks/wiki#3377] [Help / Questions]
**Question**
Hi, I am new to Wiki.js. It's already running, but I want to connect it to my Active Directory over LDAP/LDAPS. Nothing happens. I have multiple systems running using LDAPS. So I think it should work. I don't get any errors, and I can't find a logfile to search for the issue. How can I debug the problem?

**Host Info (please complete the following information):**
OS: Debian 10 (all update till today)
Wiki.js version: 2.3.81
Database engine: PostgreSQL 11.7

**Comment by NGPixel** [comment]
Nothing happens when?

**Comment by cjs1976** [comment]
Hi,

after a lot of googling and trial and error, I was able to get it up and running. It also works already over ldaps (encrypted). The only thing I am not happy with, is my search filter:

***
(&(objectcategory=person)(objectclass=user)(|(samaccountname={{username}})(mail={{username}})))
***

I have a special security group, and I want only the users mentioned in this group to be available.

Can you give me a hint how the search string shoudl look like? I made some tests with another software, but if I copy the working search string, the authentication in wiki.js doesn't work anymore.

Thanks,
Christian.

**Comment by haudamekki** [comment]
> Hi,
> 
> after a lot of googling and trial and error, I was able to get it up and running. It also works already over ldaps (encrypted). The only thing I am not happy with, is my search filter:
> 
> (&(objectcategory=person)(objectclass=user)(|(samaccountname={{username}})(mail={{username}})))
> 
> I have a special security group, and I want only the users mentioned in this group to be available.
> 
> Can you give me a hint how the search string shoudl look like? I made some tests with another software, but if I copy the working search string, the authentication in wiki.js doesn't work anymore.
> 
> Thanks,
> Christian.

I have the same problem. Any hints on this?

Thanks

Adrian

**Comment by cjs1976** [comment]
Hi Adrian,

sorry for the delay. No, I don't have any news about this. I only have 4 users, so no big deal. Maybe I find time to do some more testing...

Thanks,
Christian.
